checker/src/org/checkerframework/checker/signedness/SignednessAnnotatedTypeFactory.java

```diff
@@ -109,6 +121,14 @@ protected TreeAnnotator createTreeAnnotator() {
     }
 
     /**
+     * Provides a way to query the Constant Value Checker, which computes the values of expressions
+     * known at compile time (constant propagation and folding).
+     */
+    ValueAnnotatedTypeFactory getValueAnnotatedTypeFactory() {
```

Comment:
How about caching this value for efficiency?  Also, I think this method should be private.